### ISSUE TITLE
Missing support for mouse events

### ISSUE THREAD

**Issue by Jacalz** [fyne-io/terminal#15]
I tested the [micro](https://github.com/zyedidia/micro) editor, which I use quite frequently, and it does not work in fyneterm. Visually, it seems to looks mostly as expected with just the colour behind the line numbers missing (see comparison images below). It is however, not possible to click anywhere inside it or type anything in to it.

This is the printout to the other terminal as it is being loaded:
```
2021/05/28 19:18:48 Unknown private escape code ?2004[hl]
2021/05/28 19:19:38 Unknown private escape code ?2004[hl]
2021/05/28 19:19:38 Unknown private escape code ?2004[hl]
2021/05/28 19:20:02 Unknown private escape code ?2004[hl]
2021/05/28 19:20:02 Unknown private escape code ?2004[hl]
2021/05/28 19:20:11 Unknown private escape code ?2004[hl]
2021/05/28 19:20:11 Unknown private escape code ?2004[hl]
2021/05/28 19:20:19 Unknown private escape code ?2004[hl]
2021/05/28 19:20:19 Unknown private escape code ?2004[hl]
2021/05/28 19:20:46 Unknown private escape code ?2004[hl]
2021/05/28 19:20:46 Unrecognised Escape: 22;0;0t
2021/05/28 19:20:46 Unknown private escape code ?2004[hl]
2021/05/28 19:20:46 Unknown private escape code ?1000[hl]
2021/05/28 19:20:46 Unknown private escape code ?1002[hl]
2021/05/28 19:20:46 Unknown private escape code ?1006[hl]
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 235
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 236
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 236
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 235
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 236
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 235
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 236
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 235
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 236
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 235
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 236
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 235
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 236
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 235
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 236
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 235
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 236
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 235
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 236
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 235
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 236
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 235
2021/05/28 19:20:46 Unsupported graphics mode 48
2021/05/28 19:20:46 Unsupported graphics mode 5
2021/05/28 19:20:46 Unsupported graphics mode 235
2021/05/28 19:20:46 Unknown private escape code ?12[hl]
```

## Micro in fyneterm:
![image](https://user-images.githubusercontent.com/25466657/120020608-3eb16e00-bfea-11eb-9589-0733ef090f5b.png)

## Micro in gnome terminal:
![image](https://user-images.githubusercontent.com/25466657/120020630-4a9d3000-bfea-11eb-9e92-7e825532a362.png)

**Comment by andydotxyz** [comment]
Typing into micro works for me now, though tap-to-move-cursor does not.
Is that the same for you @Jacalz?

**Comment by Jacalz** [comment]
Yeah. Tapping inside it does not seem to work at all. Pressing on the tabs is not working.

**Comment by andydotxyz** [comment]
I think that is fixed in PR #16 :)
